packages/metrics/src/features/performance/viewRefresher.ts

```diff
@@ -26,10 +26,27 @@ import { MongoClient } from 'mongodb'
  * - One that tracks whether an update is in progress
  * - One that tracks if an uodate has been requested after the first update run started
  */
+import * as Hapi from '@hapi/hapi'
 
 let updateInProgress = false
 let nextUpdateRequested = false
 
+export async function metabaseDataRefreshHandler(
+  request: Hapi.Request,
+  h: Hapi.ResponseToolkit
+) {
+  try {
+    await refresh()
+  } catch (err) {
+    console.error(err)
+  }
```

Comment:
```suggestion
  refresh().catch(logger.error)
```
The refresh would take some time. So we should not wait for it to finish. Instead we can only trigger it on the handler.

Reply:
@Zangetsu101  Is there a downside to informing the caller when the operation is finished? I think doing that would give more control to the caller to decide if they want to do so or now and for instance send an alert forwards if it fails or retry.

Reply:
The debugging path is also something to consider here. If we at some point notice the data not updating properly, I would guess many would first go to see the scheduler logs for an error. Personally I would also prefer to have the logs there as the microservice logs are filled with HTTP metadata